static/src/js/util/url/url.js

```diff
@@ -85,6 +85,12 @@ export const decodeUrlParams = (paramString) => {
   // Decode the paramString
   const params = qs.parse(paramString, { ignoreQueryPrefix: true, parseArrays: false })
 
+  const invalidParamKeys = ['m']
```

Comment:
I think this should be pulled out into a util file, like `deprecatedParamKeys.js`, just to make it easier to find and update in the future.